data/curated/template/intro.md

```diff
@@ -7,3 +7,7 @@ directory + automatic scripts. We usually include brief introduction along the l
 --> 
```

Comment:
There's an awkward line break between 3 and 4. 
<img width="752" alt="Screenshot 2024-08-17 at 9 33 19 AM" src="https://github.com/user-attachments/assets/0d0395ef-697d-48fb-b2c8-5c890225905b">